app/controllers/profile_controller.rb

```diff
@@ -0,0 +1,25 @@
+class ProfileController < ApplicationController
+  before_action :load_user, only: [:show, :update]
+
+  def show
+  end
+
+  def update
+    if (@user).update(user_params)
+      flash[:notice] = 'Update user successfully'
+      redirect_to root_path
+    else
+      render 'edit'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.